components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -0,0 +1,160 @@
+        return (
+            <>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["starry"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img src={starryEyed} alt="starry eyed emoji" width={width || "24px"} title="starry" />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["happy"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="happy"
+                        src={happy}
+                        alt="happy emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["meh"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="meh"
+                        src={meh}
+                        alt="meh emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["crying"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="crying"
+                        src={crying}
+                        alt="crying emoji"
+                        width={width}
+                    />
+                </button>
+            </>
+        );
+    };
+    return (
+        <>
+            <h3 className="mb-4">Send Feedback</h3>
+            {selectedEmoji ? (
+                <>
+                    <div className="flex flex-col -mx-6 px-6 py-4 border-t border-b border-gray-200 dark:border-gray-800 bg-white dark:bg-gray-900">
+                        <div className="relative">
+                            <div className="absolute flex bottom-5 right-5 -space-x-3">{emojiGroup(24)}</div>
+                            <textarea
```

Comment:
Good point and I actually implement a "Thank you for your feedback" response for the error message implementation (PR incoming after this one is merged). Feel free to send a design idea for the modal!

Reply:
Maybe something like this could suffice, re-using the copy from the docs feedback widget here.

<img width="1440" alt="Modal" src="https://user-images.githubusercontent.com/120486/169098522-7438c54f-fac2-491c-959c-e4392adea6aa.png">

See [design specs](https://www.figma.com/file/HknoZvF952qXt3Sk4wLdrq/Low-friction-dashboard-form-to-collect-feedback-about-Gitpod-features-%237925?node-id=1068%3A1523) for grabbing the emoji SVG.

Reply:
Yes, will do it in a follow-up PR!

Reply:
**fyi**: Opened a follow-up issue in https://github.com/gitpod-io/gitpod/issues/10124 to keep track of this.